packages/web/components/layouts/main.tsx

```diff
@@ -182,3 +179,52 @@ const OsmosisFullLogo: FunctionComponent<{
     }}
   />
 );
+
+interface DrawerProps {
```

Comment:
Conventionally, we don't create TS types for components unless they are expected to be consumed by a caller. Please embed directly in type param.